libs/poolWorker.js

```diff
@@ -5,6 +5,23 @@ var net     = require('net');
 var MposCompatibility = require('./mposCompatibility.js');
 var ShareProcessor = require('./shareProcessor.js');
 
+loadWhitelist = function() {
+    var fs = require('fs');
+    JSON.minify = JSON.minify || require("node-json-minify");
+    
+    if (!fs.existsSync('./whitelist.json')) {
+        console.log('whitelist.json file does not exist. All workers would be accepted.');
+        return [];
+    }
+    
+    var whitelistConfig = JSON.parse(JSON.minify(fs.readFileSync("./whitelist.json", {encoding: 'utf8'})));
```

Comment:
Is there a reason to minify JSON before parsing? The parser should work the same on minified and regular files.

Reply:
You are right. The code can be simplified.